.chloggen/elasticsearchexporter-fix-event-attributes.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: elasticsearchexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Fix attributes stored with events.
```

```suggestion
note: Move the `HostID`, `ContainerID`, `K8SPodName` and `K8sNamespaceName` from the Sample to Resource.
```

Also, the service name attribute should be its own changelog entry.